lib/accounts-emails-field.js

```diff
@@ -32,28 +32,45 @@ var whitelistedServices = ['facebook', 'linkedin'];
 
 
 
-getEmailFromService = function(service_name, service){
+getEmailsFromService = function(service_name, service){
     // Picks up the email address from the service
     // NOTE: different services use different names for the email filed!!!
     //       so far, `email` and `emailAddress` were found but it may be the
     //       new names should be added to support all 3rd-party packages!
-    var email = service.email || service.emailAddress;
-    if (!email)
-        // e.g. GitHub provides a null value in the field "email" in case the email
-        //      address is not verified!
-        return {address: null, verified: false};
-
-    var verified = false;
-    // Tries to determine whether the 3rd-party email was verified
-    // NOTE: so far only for the service `google` it was found a field
-    //       called `verified_email`. But it may be that new names 
-    //       should be atted to better support all 3rd-party packages!
-    if (_.indexOf(whitelistedServices, service_name) > -1)
-        verified = true;
-    else if (service.verified_email)
-        verified = true;
-
-    return {address: email, verified: verified};
+    // Addition by @neopostmodern: Meteor developer accounts support multiple
+    //      emails themselves, rewrote to look for `emails` too and everything
+    //      must be array based then.
+
+    var emails = [];
+
+    if (service.email)
+      emails = [ { address: service.email } ];
+    if (service.emailAddress)
+      emails = [ { address: service.emailAddress } ];
+    if (service.emails)
+      emails = service.emails;
+
+    return emails.map(function (email) {
+      if (!email.address)
+          // e.g. GitHub provides a null value in the field "email" in case the email
+          //      address is not verified!
+          return {address: null, verified: false};
```

Comment:
...actually it is skipped [here](https://github.com/splendido/meteor-accounts-emails-field/blob/master/lib/accounts-emails-field.js#L95)